### ISSUE TITLE
support @JsonIgnoreProperties annotation

### ISSUE THREAD

**Issue by zyro23** [springfox/springfox-grails-integration#3]
seems `@JsonIgnoreProperties` on model classes is currently ignored.

https://github.com/springfox/springfox/issues/113 suggest to use `@JsonIgnore` instead.

however, in grails, command objects usually implement the `Validateable` trait that adds a `getErrors()` method. but you cannot annotate that getter with `@JsonIgnore` because it is injected by the trait at compile time. my common swaggered grails command object (using `swagger-servlet`, not springfox) looks like:

```groovy
@JsonIgnoreProperties("errors")
class MyCommand implements Validateable { ... }
```

**Comment by dilipkrish** [comment]
@zyro23 thanks for reporting issues. Thesis great feedback!

**Comment by dilipkrish** [comment]
@zyro23 addressed via fixing #6
